README.md

```diff
@@ -36,16 +36,21 @@ Usage of goimports-reviser:
         File path to fix imports(ex.: ./reviser/reviser.go). Required parameter.
```

Comment:
`File path to fix imports(ex.: ./reviser/reviser.go). Required parameter.` -> `File path to fix imports(ex.: ./reviser/reviser.go). Optional parameter.`

Reply:
Thanks, will fix it

Reply:
Sorry, I have just realised that it's already fixed in commit bf5ff2e. Please take a look.
I amended the original commit to increase the test coverage and to fix the "Required" on 05/Nov/2021

Reply:
not in the branch I believe
![image](https://user-images.githubusercontent.com/7884395/142609555-37e53020-3d62-4393-a658-acd70ac7ce48.png)

Reply:
Thanks, it looks like github misbehaves with "commit --amend && push -f", so I had to create a new branch and create a new pull request (#67). Please take a look at that, which is still the same code as 11 days ago (bf5ff2e).
I am going to close this PR #65